app/components/molecules/perk-info/perk-info.styles.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+
+const desktopMediaQuery = '@media (min-width: 768px)';
```

Comment:
Didn't we use this media query string somewhere else? so you don't create it again 👀

Reply:
I had a way to do this but I ended up not liking it. I just refactored it, though, and I think it's a cleaner approach.